components/infra-proxy-service/server/roles.go

```diff
@@ -55,27 +55,12 @@ func (s *Server) CreateRole(ctx context.Context, req *request.CreateRole) (*resp
 		runList = []string{}
 	}
 
-	var defAtt interface{}
-	var ovrAtt interface{}
-
-	defIn := req.DefaultAttributes
-	if defIn == "" {
-		defIn = "{}"
-	}
```

Comment:
Is this case now covered by `StructToJSON` ? Would be nice to see some unit tests that show that. Also, I note that you have `DefaultIfEmpty` defined but that is not used anywhere. https://github.com/chef/automate/blob/d1fbf37902cc25fa425a89664c244d0b81d3833f/components/infra-proxy-service/server/proxy.go#L140
Again wondering if `StructToJSON` covers that?

Reply:
Thanks @msorens will update it.